### DISCUSSION TITLE
Welcome to playspace Discussions!

### DISCUSSION THREAD

**Discussion by dt-r** [dt-r/playspace#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Upvote the FHIR Resources that should be profiled in Release 1 of AU Core for your products so we can work on what we need to
  * Ask questions you’re wondering about.
  * Share ideas on future work and other feature requests so we can discuss as a community.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dt-r** [comment]
### GitHub Discussions vs GitHub Issues - when to use for AU Core

#### When to use GitHub Discussions

GitHub Discussions is where we will conduct polls and other voting, and is best suited as a tool to share questions, ideas, conversations, requests for comment (RFC), resource planning, and announcements.

Ideas on future work or feature requests should start as discussions.

#### When to use GitHub Issues

GitHub Issues are used to track executable pieces of work with a defined start and end point. These can include new features, fixing bugs, general updates, and tracking for epics and sprints, among other things. Often an issue started life as a discussion and at a certain point we have enough consensus to create an issue and proceed.

[We can create an issue directly from a discussion](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion), which means you can attach your notes and decisions that led to a piece of a work directly in an Issue.

We can also convert issues into discussions if something needs further discussion.

**Comment by brettesler-ext** [comment]
more comments from me
